packages/api-nest/src/queue/services/video-extractor.service.spec.ts

```
@@ -0,0 +1,106 @@
+
+        expect(result.title).toBeDefined()
+        expect(result.author).toBeDefined()
+        expect(result.videoId).toBe('dQw4w9WgXcQ')
+        expect(result.platform).toBe('youtube')
+        expect(result.embedUrl).toBe(
+          'https://www.youtube.com/embed/dQw4w9WgXcQ',
+        )
+        expect(result.duration).toBeGreaterThan(0)
+      } catch (error) {
+        // If network request fails in test environment, skip
+        console.warn('Video extraction test skipped due to network error:', error)
```

ESLint

[error] Replace `'Video·extraction·test·skipped·due·to·network·error:',·error` with `⏎··········'Video·extraction·test·skipped·due·to·network·error:',⏎··········error,⏎········`

(prettier/prettier)